src-tool/Pact/Analyze/Model/Text.hs

```diff
@@ -229,7 +235,18 @@ showEvent ksProvs tags event = do
                   argLines = case length vids of
                                0 -> []
                                _ -> displayVids showArg ++ [emptyLine]
-              in headerLine : argLines
+
+                  vals :: Map VarId Unmunged
+                  vals = Map.mapMaybe (\case Located _ (n, (EType SDecimal, AVal p sval)) ->
+                                               case unliteralS $ mkS p sval of
+                                                 Just v | Pact.decimalPlaces (toPact decimalIso v) == 255 -> Just n
```

Comment:
If 255 is a "Pact language constant", can we put that into a module within the Pact codebase and then import that here into FV, so that it's insulated from these types of concrete decisions?

Reply:
`Data.Decimal` uses `m * 10^(-e)` with `m :: Integer` and `e :: Word8` as the underlying representation.  How about using `maxBound :: Word8` ?

Reply:
i'm fine with `maxBound` here

Reply:
@jwiegley are u also ok with the `maxBound` approach?

Reply:
Yes, `maxBound` sounds like a good choice.